liquibase-standard/src/main/java/liquibase/exception/UnsupportedChecksumVersionException.java

```diff
@@ -1,10 +1,15 @@
 package liquibase.exception;
 
+import liquibase.util.LiquibaseUtil;
+
 public class UnsupportedChecksumVersionException extends RuntimeException {
 
     private static final long serialVersionUID = -229400973681987065L;
 
     public UnsupportedChecksumVersionException(int i) {
-        super("Unsupported Checksum version: " + i);
+        super(String.format("Liquibase detected an unknown checksum calculation version %s in the DatabaseChangelog table. " +
+                "This can be caused by a previous operation on this changelog with a newer version than your current %S release. " +
+                "Please update your current Liquibase, or clear checksums on changesets you are certain you need " +
+                "to deploy.", i, LiquibaseUtil.getBuildVersionInfo()));
```

Comment:
I don't think we should even mention clear checksums.

```suggestion
        super(String.format("Liquibase detected an unknown checksum calculation version %s in the DatabaseChangelog table. " +
                "This can be caused by performing operations on this changelog/database with a newer version of Liquibase than your current %S release. " +
                "Please update your current Liquibase to perform this operation against this database..", i, LiquibaseUtil.getBuildVersionInfo()));
```

Reply:
This error message was created based on the suggestion from DAT-14876 . Maybe we can discuss it there and then update the message? Meanwhile I"ll make it simple, thanks!